masonry/src/widget/sized_box.rs

```diff
@@ -82,6 +82,20 @@ impl Padding {
     /// A padding of zero for all edges.
     pub const ZERO: Padding = Padding::all(0.);
 
+    /// An padding which can be detected using, allowing parent views to determine if the padding is under user control.
```

Comment:
It looks like a word is missing after "using".

Reply:
Also, "A padding" rather than "An ..."